salt/haproxy/errors/503-velum_database_not_ready.http

```diff
@@ -0,0 +1,73 @@
+HTTP/1.0 503 Service Unavailable
+Cache-Control: no-cache
+Connection: close
+Content-Type: text/html
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>SUSE CaaS Platform is initializing</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <style>
+    body {
+      background-color: #003258;
+      color: #2E2F30;
+      text-align: center;
+      font-family: Roboto, "Helvetica Neue", Helvetica, Arial, sans-serif;
+      margin: 0;
+    }
+
+    div.dialog {
+      width: 95%;
+      max-width: 33em;
+      margin: 4em auto 0;
+    }
+
+    div.dialog > div {
+      border-bottom: 1px solid #BBB;
+      border-top: #00c081 solid 4px;
+      border-top-left-radius: 9px;
+      border-top-right-radius: 9px;
+      background-color: #00243E;
+      box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+    }
+
+    div.dialog > div > p {
+      color: #fff;
+    }
+
+    h1 {
+      font-size: 100%;
+      color: #fff;
+      line-height: 1.5em;
+      font-weight: 500;
+      font-size: 18px;
+    }
+
+    div.dialog > p {
+      margin: 0 0 1em;
+      padding: 1em;
+      background-color: #F7F7F7;
+      border: 1px solid #CCC;
+      border-right-color: #999;
+      border-left-color: #999;
+      border-bottom-color: #999;
+      border-bottom-left-radius: 4px;
+      border-bottom-right-radius: 4px;
+      border-top-color: #DADADA;
+      color: #333333;
+      box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+    }
+  </style>
+</head>
+
+<body>
+  <div class="dialog">
+    <div>
+      <h1>SUSE CaaS Platform is initializing</h1>
+    </div>
+    <p>If you just finished the installation, it may take some seconds for the database to be available. Should the error persist check the logs for more information.</p>
+  </div>
+</body>
+</html>
```

Comment:
We cannot create this file with salt. It needs to be installed with some package, otherwise on the very first run of the admin node this file won't exist.

Reply:
i wanted to use the one we have in `velum/public` but that doesnt contain the headers, but maybe it works as well?

Reply:
@ereslibre we could simply add the header in https://github.com/kubic-project/velum/blob/master/public/503_database_not_ready.html but then we also need to mount the velum source directory into the salt-master container. do you think that would be worth it?